fe1-web/src/features/digital-cash/reducer/DigitalCashReducer.ts

```diff
@@ -31,6 +31,12 @@ export interface DigitalCashReducerState {
    * this public key hash in one or more of their TxOuts or their TxIns
    */
   transactionsByPubHash: Record<string, string[]>;
+
+  /**
+   * A mapping between public key hashes and an array of hash of transactions in which the public
+   * key is involved, as input or output.
+   */
+  transactionsByInvPubHash: Record<string, string[]>;
```

Comment:
I'm a bit confused by the naming. What does `Inv` stand for ?

Reply:
It's involved, but it may be a bit awkward. It's just that `transactionsByInvolvedPubHash` felt a bit long. Do you have any other suggestion?

Reply:
would it make sense to have `allTransactionsByHash` and `userTransactionsByHash` ?

Reply:
Yes it would maybe make more sense, I'll look into this!